koku/masu/util/aws/aws_post_processor.py

```diff
@@ -31,6 +37,29 @@ def handle_user_defined_json_columns(data_frame, columns, column_prefix):
     return column_dict.apply(json.dumps), unique_keys
 
 
+def create_enabled_categories(schema, enabled_keys):
+    enabled_keys_model = AWSEnabledCategoryKeys
+    ctx = {"schema": schema, "enabled_categories": enabled_keys}
+    if not enabled_keys:
+        LOG.info(log_json(msg="no categories found, skipping category enablement", context=ctx))
+        return
+    with schema_context(schema):
+        LOG.info(log_json(msg="searching for new category keys", context=ctx))
+        new_keys = list(set(enabled_keys) - {k for k in enabled_keys_model.objects.values_list("key", flat=True)})
+        if new_keys:
+            LOG.info(log_json(msg="creating enabled key records", context=ctx))
+            for batch_num, new_batch in enumerate(batch(new_keys, _slice=500)):
+                batch_size = len(new_batch)
+                LOG.info(
+                    log_json(msg="create batch", batch_number=(batch_num + 1), batch_size=batch_size, context=ctx)
+                )
+                for ix in range(batch_size):
+                    new_batch[ix] = enabled_keys_model(key=new_batch[ix])
+                enabled_keys_model.objects.bulk_create(new_batch, ignore_conflicts=True)
+        else:
+            LOG.info(log_json(msg="no enabled keys added", context=ctx))
```

Comment:
nitpick: but we should avoid all the logic buried in an `if` statement and the `else` just logs a message
```suggestion
        if not new_keys:
            LOG.info(log_json(msg="no enabled keys added", context=ctx))
            return
            
        LOG.info(log_json(msg="creating enabled key records", context=ctx))
        for batch_num, new_batch in enumerate(batch(new_keys, _slice=500)):
            batch_size = len(new_batch)
            LOG.info(
                log_json(msg="create batch", batch_number=(batch_num + 1), batch_size=batch_size, context=ctx)
            )
            for ix in range(batch_size):
                new_batch[ix] = enabled_keys_model(key=new_batch[ix])
            enabled_keys_model.objects.bulk_create(new_batch, ignore_conflicts=True)
```